libraries/standard/mqtt/cbmc/proofs/Makefile-project-targets

```diff
@@ -0,0 +1 @@
+../../../../../tools/aws-templates-for-cbmc-proofs/template-for-repository/proofs/Makefile-project-targets
```

Comment:
Per offline conversation we shall instruct users to run setup.py in the template submodule.